app-tasks/rf/src/rf/utils/batch.py

```diff
@@ -0,0 +1,16 @@
+import boto3
+
+def get_latest_job_revision(job_definition_name):
```

Comment:
Do we still have need for this given that Boto3 seems to default to the most recent revision when no revision number is supplied?

Reply:
Yeah, I'll remove this now -- I didn't have a good way to test it because the one batch job mentioned here didn't exist, but you've confirmed it with your PR in the deployment repository.